tensorzero-core/src/providers/mod.rs

```diff
@@ -15,6 +15,7 @@ pub mod groq;
 pub mod helpers;
 pub mod helpers_thinking_block;
 pub mod hyperbolic;
+pub mod llama;
```

Comment:
Same here and in similar mentions, let's prefer `llama_api` to just `llama` everywhere.